examples/velero_at_a_br_at_other.md

```diff
@@ -0,0 +1,51 @@
+# Velero at project A, backup and restore at other projects
+
+This scenario is introduced in [issue 4806](https://github.com/vmware-tanzu/velero/issues/4806).
+
+Assume the following...
+
+- Project A [project-a]: The project where the Velero's service account is located, and the Velero service account is granted to have enough permission to do backup and restore in the other projects.
+- Project B [project-b]: The GCP project we want to restore TO.
+- Project C [project-c]: The GCP project we want to restore FROM.
```

Comment:
you mean backup?

Reply:
Reused the word from https://github.com/vmware-tanzu/velero-plugin-for-gcp/blob/main/examples/gcp-projects.md, but the backup sounds more appropriate here.